docs/modules/plugin/pages/goals/http.adoc

```diff
@@ -0,0 +1,66 @@
+[[http-goal]]
+= http: HTML life preview
+
+Converts Asciidoc documents and starts an embedded http server to access content from the generated output directory.
+
+Modified sources will be updated on change similarly to how the xref:goals/auto-refresh.adoc[auto-refresh goal] works.
+While at the same time, HTML contents will be automatically refreshed on the web browser without need for manual steps.
+Just open the file through the provided url that will appear in the console and write.
+
+[IMPORTANT]
+====
```

Comment:
If an admonition is only a single paragraph, you don't need the block delimiters around it.

Reply:
Fixed this and a couple of more cases.
I mantained one WARNING that has 4 senteces and is intertwined with the dlist of the parameters. I think the block makes reading it easier in the source.